.gitignore

```diff
@@ -1 +1,13 @@
+*~
 /third_party/llvm
+.ninja*
```

Comment:
might be nice to run cmake from a build directory so it doesn't dump all this stuff in the root.